pkg/sql/export.go

```diff
@@ -117,10 +125,21 @@ func (ef *execFactory) ConstructExport(
 		}
 	}
 
+	// Check whenever compression is expected and extract compression codec name in case
+	// of positive result
+	codecName := ""
```

Comment:
IMO  could just go straight to the enum here and skip the string.
```
var codec execinfrapb.FileCompression
if ... {
  codec = execinfrapb.FileCompression_Gzip
}
```

Reply:
done